web/src/containers/activity/Schedule.js

```diff
@@ -41,7 +41,7 @@ const Schedule = ({ activity, activityIndex, setEndDate, setStartDate }) => {
             {stateDateToDisplay(activity.plannedStartDate)}
           </p>
           <p>
-            <strong>Planned end date:</strong>{' '}
+            <strong>Target completion date:</strong>{' '}
             {stateDateToDisplay(activity.plannedEndDate)}
```

Comment:
Question: Does it make sense to update the activity date property names to 'startDate' and 'endDate', or something similar? @pphillips-fearless @mgwalker